7.2_kfc_merge_files_datatype_has_dicts/kfc.yaml

```diff
@@ -1,29 +1,60 @@
-amlIdentifier:
+moduleIdentifier:
   moduleName: Merge every n files together cdd3b0ba
-  version: 0.0.1
+  moduleVersion: 0.0.1
+  namespace: test/test
 inputs:
-- name: InputFolder
+- name: InputFolder with single simple type
+  type: String
```

Comment:
Let's remove the type since it's optional

Reply:
As discussed, let's keep type required.